recirq/otoc/loschmidt/tilted_square_lattice/gen-small-v1.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+
+import cirq
+from recirq.otoc.loschmidt.tilted_square_lattice import \
+    get_all_tilted_square_lattice_executables
+
+FN = 'loschmidt.tilted_square_lattice.small-v1.json.gz'
```

Comment:
What is this?  Avoid abbreviations, especially when ambiguous (filename vs function).
Suggest OUTPUT_FILENAME or similar?

Reply:
changed.

also I think at some point this filename should be fetched maybe from the `algorithmic_benchmarking_library` "card catalog" module but I haven't completely thought through how that would work.